frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -43,6 +43,7 @@ export enum GUARDED_FEATURES {
   OCS_MULTUS = 'OCS_MULTUS',
   OCS_ARBITER = 'OCS_ARBITER',
   OCS_KMS = 'OCS_KMS',
+  OCS_KMS_VAULT_SERVICE_ACCOUNT = 'OCS_KMS_VAULT_SERVICE_ACCOUNT',
```

Comment:
Nits, very minor...
should be `ODF_KMS_VAULT_SERVICE_ACCOUNT`

Reply:
ack